QUICKFIX_LICENSE

```diff
@@ -1,6 +1,6 @@
 The QuickFIX Software License, Version 1.0
 
-Copyright (c) 2001-2010 quickfixengine.org  All rights
+Copyright (c) 2001-2020 quickfixengine.org  All rights
```

Comment:
Idk if this should be changed in the quickfix license

Reply:
I'll revert